src/misc/sw-template.js

```diff
@@ -299,7 +299,7 @@ function WebpackServiceWorker(params) {
 }
 
 function addAllNormalized(cache, requests) {
-  return Promise.all(requests.map(fetch)).then((responses) => {
+  return Promise.all(requests.map((request) => fetch(request))).then((responses) => {
```

Comment:
Yes, it threw an error in Chrome Canary and the service worker failed to install. I know it doesn't look as nice and it works in Chrome, but technically Chrome Canary is right - `fetch` gets passed an invalid argument as parameter 2.